geotrek/trekking/tests/test_parsers.py

```diff
@@ -563,3 +573,543 @@ def test_create(self, mocked_head, mocked_get):
+        self.assertEqual(trek.ambiance_fr, 'La description détaillée en français.')
+        self.assertEqual(trek.ambiance_en, 'The longer description in english.')
+        expected_fr_description = (
+            '<p>Départ : du parking de la Chapelle Saint Michel </p>'
+            '<p>1/ Suivre le chemin qui part à droite, traversant le vallon.</p>'
+            '<p>2/ Au carrefour tourner à droite et suivre la rivière</p>'
+            '<p>3/ Retour à la chapelle en passant à travers le petit bois.</p>'
+            '<p>Ouvert toute l\'année</p>'
+            '<p>Fermeture exceptionnelle en cas de pluie forte</p>'
+            '<p>Suivre le balisage GR (blanc/rouge) ou GRP (jaune/rouge).</p>'
+            '<p>Montée en télésiège payante. 2 points de vente - télésiège Frastaz et Bois Noir.</p>'
+        )
+        self.assertEqual(trek.description_fr, expected_fr_description)
+        expected_en_description = (
+            '<p>Start: from the parking near the Chapelle Saint Michel </p>'
+            '<p>1/ Follow the path starting at right-hand, cross the valley.</p>'
+            '<p>2/ At the crossroad turn left and follow the river.</p>'
+            '<p>3/ Back to the chapelle by the woods.</p>'
+            '<p>Open all year long</p>'
+            '<p>Exceptionally closed during heavy rain</p>'
+            '<p>Follow the GR (white / red) or GRP (yellow / red) markings.</p>'
+            '<p>Ski lift ticket office: 2 shops - Frastaz and Bois Noir ski lifts.</p>'
+        )
+        self.assertEqual(trek.description_en, expected_en_description)
+        self.assertEqual(trek.advised_parking_fr, 'Parking sur la place du village')
+        self.assertEqual(trek.advised_parking_en, 'Parking sur la place du village')
+        self.assertEqual(trek.departure_fr, 'Sallanches')
+        self.assertEqual(trek.departure_en, 'Sallanches')
+        self.assertEqual(trek.access_fr, 'En voiture, rejoindre le village de Salanches.')
+        self.assertEqual(trek.access_en, 'By car, go to the village of Sallanches.')
+        self.assertEqual(trek.access_it, 'In auto, andare al villaggio di Sallances.')
+        self.assertEqual(len(trek.source.all()), 1)
+        self.assertEqual(trek.source.first().name, 'Office de tourisme de Sallanches')
+        self.assertEqual(trek.source.first().website, 'https://www.example.net/ot-sallanches')
+
+        self.assertTrue(trek.difficulty is not None)
+        self.assertEqual(trek.difficulty.difficulty_en, 'Level red – hard')
+
+        self.assertTrue(trek.practice is not None)
+        self.assertEqual(trek.practice.name, 'Pédestre')
+
+        self.assertEqual(trek.networks.count(), 2)
+        networks = trek.networks.all()
+        self.assertIn('Hiking itinerary', [n.network for n in networks])
+        self.assertIn('Pedestrian sports', [n.network for n in networks])
+
+        from geotrek.common.models import Attachment
```

Comment:
Is there a reason not to have this at the top ?

Reply:
Oops, I missed that. Removed.